ui/articles-aem-renderer/src/main/java/org/cru/godtools/articles/aem/fragment/AEMArticleItemFragment.java

```diff
@@ -138,21 +147,62 @@ private String injectCss(@NonNull final String content) {
 
     // endregion WebView content
 
+    //region WebClient
+
     private class AEMWebViewClient extends WebViewClient {
         @Nullable
         @Override
         public WebResourceResponse shouldInterceptRequest(WebView view, String url) {
-            if (url.endsWith(".jpg")) {
-                //TODO: Load local Image WebResource
-            } else if (url.endsWith(".css")) {
-                //TODO: Load CSS WebResource
+            String extension = MimeTypeMap.getFileExtensionFromUrl(url);
+
+            if (extension.isEmpty()) {
+                return super.shouldInterceptRequest(view, url);
```

Comment:
We may want to implement a "Not Found" response behavior when we don't recognize the resource that we are trying to load.

Reply:
@frett I think I can just remove it with the switch statement in place it really isn't needed.